genome_kit/_apply_variants.py

```diff
@@ -53,6 +53,44 @@ def check_variants_list(reference_genome, variants):
 
     return variants
 
+def _reverse_tuples_in_alignment(alignment):
+    result = []
+    tmp = []
+
+    for i in alignment:
+        if isinstance(i, tuple):  # If the item is a tuple, add it to the temp list
+            tmp.append(i)
+        else:
+            if tmp:
```

Comment:
I prefer not to rely on falsey values.
```suggestion
            if len(tmp) > 0:
```